### PR TITLE
improved the op lowering guideline

### PR DESCRIPTION
**Pull request by miladm**

### CODE REVIEW

**Review by JackCaoG — CHANGES_REQUESTED** [review]
nits

File: OP_LOWERING_GUIDE.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ All file mentioned below lives under the `xla/torch_xla/csrc` folder, with the e
...
 5. `tensor.h` contains the `XLATensor` declarations. These declarations are usually a one to one mapping of the `at::Tensor` nodes we declared in `XLANativeFunctions.h`
 6. `tensor_methods.cpp` contains the implementation of `XLATensor node` defined in `tensor.h`. We constructed the corresponding `ir::op` from the parameter’s `ir::Value` and wrapped it inside a `XLATensor`. Ir stands for intermediate representation.
-7. `ops/` directory contains all `ir::ops` declaration and definition. Smaller nodes can be put in `ops/ops.h/.cpp`. More complicated nodes can be put into a separate file. All ops inherit from `ir::ops::Node` and provide a way to lower input `ir::Value` to a sequence of `XlaOp`.
+7. `ops/` directory contains all `ir::ops` declaration and definition. Smaller nodes can be put in `ops/ops.h/.cpp`. More complicated nodes can be put into a separate file. All ops inherit from `ir::ops::Node` and provide a way to lower input `ir::Value` to a sequence of `XlaOp`. Verify that the op class name is consistent with what's available in `pytorch/aten/src/ATen/core/aten_interned_strings.h`.
```
[the remark this answers is not in the document]
Comment (bdhirsh) [reply OP_LOWERING_GUIDE.md:25]: I dunno what the history is, but it looks like the symbols from both of those files are lumped together here: https://github.com/pytorch/pytorch/blob/fd13e925ecdd20294c5f28dd5cfa367fc3048c53/aten/src/ATen/core/interned_strings.h#L365.

So as long as the symbol is prefixed with `aten` in either of those files, it looks like it'll get a symbol. `aten_interned_strings.h` seems like the cleaner place to put it, but I also see plenty of aten symbols in the other file like you said.

So given all of that, probably wouldn't hurt to mention both files in the doc.

[the remark this answers is not in the document]
Comment (miladm) [reply OP_LOWERING_GUIDE.md:25]: Addressed.

File: OP_LOWERING_GUIDE.md [hunk 2]
```diff
@@ -80,3 +80,5 @@ at::Tensor & wrapper_Scalar_lerp_(at::Tensor & self, const at::Tensor & end, con
 ```
 
 `lerp_.Scalar` will use our `lerp.Scalar` implementation without us providing explictly lowering.
+
+For each node we need to pass a `ir::OpKind` which is usually in the format of `at::aten::op_name` ([link to example](https://github.com/pytorch/xla/blob/5ce99bff336325feb41a982dc80299fb53166b29/torch_xla/csrc/ops/var_mean.cpp#L36)). You can find the `OpKind` definition in [aten_interned_strings.h](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/aten_interned_strings.h) or [interned_strings.h](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/interned_strings.h). If the aten symbol is missing, you can submit a PR like this [link](https://github.com/pytorch/pytorch/pull/36851) to PyTorch.
```
Comment (JackCaoG) [inline OP_LOWERING_GUIDE.md:84]: nit, "an `ir::OpKind`". You can also do , 
```
Here is an [example] (https://github.com/pytorch/xla/blob/5ce99bff336325feb41a982dc80299fb53166b29/torch_xla/csrc/ops/var_mean.cpp#L36)).
```

**Review by JackCaoG — CHANGES_REQUESTED** [review]

File: OP_LOWERING_GUIDE.md (hunk 2 above)
Comment (miladm) [reply OP_LOWERING_GUIDE.md]: Done.

File: OP_LOWERING_GUIDE.md [hunk 3]
```diff
@@ -80,3 +80,5 @@ at::Tensor & wrapper_Scalar_lerp_(at::Tensor & self, const at::Tensor & end, con
 ```
 
 `lerp_.Scalar` will use our `lerp.Scalar` implementation without us providing explictly lowering.
+
+For each node we need to pass an `ir::OpKind`. Here is an [example] (https://github.com/pytorch/xla/blob/5ce99bff336325feb41a982dc80299fb53166b29/torch_xla/csrc/ops/var_mean.cpp#L36)). You can find the `OpKind` definition in [aten_interned_strings.h](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/aten_interned_strings.h) or [interned_strings.h](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/interned_strings.h). If the aten symbol is missing, you can submit a PR like this [link](https://github.com/pytorch/pytorch/pull/36851) to PyTorch.
```
Comment (JackCaoG) [inline OP_LOWERING_GUIDE.md:84]: nit, 
```
you can submit a PR like [this](https://github.com/pytorch/pytorch/pull/36851)
```

**Review by JackCaoG — CHANGES_REQUESTED** [review]

**Review by JackCaoG — APPROVED** [review]
Thanks @miladm !

**Pull request merged**